pkg/execution/state/state.go

```diff
@@ -31,6 +32,17 @@ const (
 type Identifier struct {
 	WorkflowID uuid.UUID `json:"workflowID"`
 	RunID      ulid.ULID `json:"runID"`
+	// Key represents a unique idempotency key used to deduplicate this
+	// workflow run amongst other runs for the same workflow.
+	Key string `json:"key:"`
```

Comment:
Is it correct that `json:"key:"` should contain the semi-colon? Just looked like a potential typo

Reply:
Def a typo — appreciate it!